client/components/Flights.js

```diff
@@ -36,7 +37,7 @@ const Flights = ({ currentUser, currentTrip }) => {
         'https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices/browsequotes/v1.0/US/USD/en-US/MSY-sky/BNA-sky/2020-09-10',
         {
           headers: {
-            'x-rapidapi-key': '0bab63c571mshc3c168e88ae6b44p14c63fjsn0a3bbb552a96',
```

Comment:
Rebase to remove this commit. You don't want your api key published where everyone can see it.